scripts/build/bootstrap/incremental_build_util.py

```diff
@@ -190,28 +227,29 @@ def find_snapshot_id(ec2_client, repository_name, project, pipeline, platform, b
                     snapshot_id = snapshot['SnapshotId']
     return snapshot_id
 
-def create_volume(ec2_client, availability_zone, repository_name, project, pipeline, branch, platform, build_type, disk_size, disk_type):
-    # The actual EBS default calculation for IOps is a floating point number, the closest approxmiation is 4x of the disk size for simplicity
+
+def create_volume(ec2_client, availability_zone, repository_name, project, pipeline, branch, platform, build_type,
+                  disk_size, disk_type):
     mount_name = get_mount_name(repository_name, project, pipeline, branch, platform, build_type)
-    pipeline_and_branch = get_pipeline_and_branch(pipeline, branch) 
     parameters = dict(
-        AvailabilityZone = availability_zone,
+        AvailabilityZone=availability_zone,
         VolumeType=disk_type,
-        TagSpecifications= [{
+        TagSpecifications=[{
             'ResourceType': 'volume',
             'Tags': [
-                { 'Key': 'Name', 'Value': mount_name },
-                { 'Key': 'RepositoryName', 'Value': repository_name},
-                { 'Key': 'Project', 'Value': project },
-                { 'Key': 'Pipeline', 'Value': pipeline },
-                { 'Key': 'BranchName', 'Value': branch },
-                { 'Key': 'Platform', 'Value': platform },
-                { 'Key': 'BuildType', 'Value': build_type }, 
-                { 'Key': 'PipelineAndBranch', 'Value': pipeline_and_branch }, # used so the snapshoting easily identifies which volumes to snapshot
```

Comment:
When this gets merged in we'll need to update the snapshot policy to replace PipelineAndBranch with Pipeline and BranchName.

Reply:
Is PipelineAndBranch used when creating the snapshot? I didn't realize it so I delete it here because I didn't see it being used anywhere when finding snapshot in this script. 

I can add PipelineAndBranch back if it's still used somewhere.

Reply:
yeah, should be added back because the lifecycle manager doesnt let you do complex operations with the tags, it basically has an "OR" between the tags, so I did the "AND" in the tagged itself.

Reply:
It might have been a stub for something in the future? The `find_snapshot_id` function on line 255 doesn't use it, and the volumes already have a `Pipeline` and `BranchName` tag. The snapshot policy just copies the tags from the volume.

Reply:
Yes the policy was using PipelineAndBranch tag for the snapshots. I reviewed the policy and we'll need to add it back. 

The policy will be applied to EBS volumes with any of the following tags. So if we use Pipeline and BranchName separately it will snapshot any volume with the 'default' pipeline tag for example. So using the PipelineAndBranch tag allows us to isolate the default_<branch_name> volume. 

It's not specifically being used by this script, but it will set it when creating a new volume and this is what's used by the snapshot policy.